templates/deploy-to-existing-kubernetes-cluster.properties.json

```diff
@@ -32,11 +32,15 @@
     "assets": [
         {
             "type": "file",
-            "path": "k8s/deployment.yml"
+            "path": "k8s/deployment.yml",
+            "destinationPath": "manifests/deployment.yml",

```

Comment:
What will happen if the repo already contains these files?

Reply:
It would throw an error like this: 
```error
The file manifests/deployment.yml already exists in the branch master of the GitHub repository ds-ms/goSampleApp.
```